app/test-utils/test-environment-builder.ts

```diff
@@ -293,81 +336,108 @@ export class TestEnvironmentBuilder<
           { runId, prismaSchemaPath: "prisma/schema.prisma" },
         );
         const dbUrl = `postgresql://postgres:pass@${DB_HOST}:5432/${rsp.dbName}`;
-
         const db = new PrismaClient({
           datasources: {
             db: { url: dbUrl },
           },
         });
-
         addTeardownFunction(async () => {
           await db.$disconnect();
         });
-
         return db;
       },
       ...this.lateInit,
     });
   }
 
   // TODO: Create a withWorker helper ;)
-  withTimeSkippingTemporal(): TestEnvironmentBuilder<
+  withTimeSkippingTemporal(workflowsPath?: string): TestEnvironmentBuilder<
```

Comment:
I remember you mentioning that we don't use time skipping. Time skipping instances can't be shared between tests cause the time is a global.

Reply:
We don't, I just never change the name "withTimeSkippingTemporal" to "withLocalTemporal". Under the hood it's using a local temporal instance.